conf/application.conf

```diff
@@ -140,6 +140,8 @@ feature-switch {
   version-2 {
     enabled = true
   }
+
+  is-jacobs-new-thing = true
```

Comment:
You'll want to default to false if you want the default behaviour to be the old behaviour.